fast_forward/universe_handler.py

```diff
@@ -36,6 +54,19 @@ def __init__(self, mol_names, *args, **kwargs):
         mol_idxs_names = [(idx, mol_name) for idx, mol_name in self.mol_idxs_by_name.items()]
         self.normal_order = mol_idxs_names.sort()
 
+        istraj = args[1].casefold().endswith(('trr', 'xtc'))
+        
+        # Required for unwrapping
+        if istraj:
+            self.pbc = np.array([ dim2lattice(*f.dimensions) for f in self.trajectory ])
+            self.coordinates = self.trajectory.coordinate_array.astype('float64')
+        else:
+            self.pbc = dim2lattice(self.dimensions)[None]
+            self.coordinates = self.atoms.positions[None].astype('float64')
+            
+        self.ipbc = 2 * np.pi * np.linalg.inv(self.pbc) 
+        self.reciprocal = (self.coordinates @ self.ipbc)
+        
```

Comment:
The pbc_complete method is not needed anymore, so it can be removed. Same goes for the hidden self.__pbc_completed attribute unless we want to set it when the new code is run. I think that would be a good idea